spec/javascripts/calculator-spec.js

```diff
@@ -0,0 +1,38 @@
+describe ("calculator", function() {
+  it("should return a calculator object", function() {
+    expect(calculator()).toBeDefined();
```

Comment:
calculator() is not a function - it is an object literal.  You would call calculator.add(x,y).  That is why your tests are throwing that error.